functions.php

```diff
@@ -36,3 +34,104 @@
 		require_once $filepath;
 	}
 	unset( $file, $filepath );
+
+	function yt_videodetails( $video_id ) {
```

Comment:
Do not store everything in `functions.php`. Instead, make structure followed structure in LA and PAP.

Reply:
Removed, used for testing content filter